### ISSUE TITLE
py37 compatibility

### ISSUE THREAD

**Issue by TK-21st** [chungheng/neural#4]
Compilation raises unexpected error when compiling `OTP` in `py37`.
All packages are installed with default pip wheel.

Unfortunately the printout is gone but it's related to codegen for `np.cbrt`.

It seems like in `py37` the cuda codegen for `np.cbrt` is wrong and the word `np` shows up in the generated kernel and the `cbrt` call that is supposed to be in the kernel is missing.

**Comment by TK-21st** [comment]
This seems to be an issue with `dis.get_instructions()` for `np.cbrt`

In PY36 this gives:
```
Instruction(opname='LOAD_ATTR', opcode=106, arg=14, argval='cbrt', argrepr='cbrt', offset=152, starts_line=None, is_jump_target=False)
```

In PY37 this gives:
```
Instruction(opname='LOAD_METHOD', opcode=160, arg=14, argval='cbrt', argrepr='cbrt', offset=152, starts_line=None, is_jump_target=False),
```

**Comment by TK-21st** [comment]
Also `CALL_METHOD` need to be handled. See https://docs.python.org/3/library/dis.html#opcode-LOAD_METHOD and below

**Comment by TK-21st** [comment]
Will be closed by #17

**Comment by TK-21st** [comment]
#17 closed by #25
